packages/@ionic/cli/src/commands/package/build.ts

```diff
@@ -318,7 +315,7 @@ if you do not wish to download ${input('apk')}.
       ['Commit', strong(`${build.commit.sha.substring(0, 6)} ${build.commit.note}`)],
       ['Target Platform', strong(build.stack.friendly_name)],
       ['Build Type', strong(build.build_type)],
-      ['Artifact Type(s)', strong(`${artifactTypes}`)],
```

Comment:
I changed this and the above code, since `AAB` or `DSYM` may not always be available by default.